yb-voyager/cmd/importData.go

```diff
@@ -197,8 +197,10 @@ func getYBServers() []*tgtdb.Target {
 
 	if loadBalancerUsed { // if load balancer is used no need to check direct connectivity
 		utils.PrintAndLog(LB_WARN_MSG)
-		parallelImportJobs = 2 * len(targets)
-		utils.PrintAndLog("Using %d parallel jobs by default. Use --parallel-jobs to specify a custom value", parallelImportJobs)
+		if parallelImportJobs == -1 {
+			parallelImportJobs = 2 * len(targets)
+			utils.PrintAndLog("Using %d parallel jobs by default. Use --parallel-jobs to specify a custom value", parallelImportJobs)
```

Comment:
Nit: Remove this print line from here
Also remove similar line from line 328
Add this line just once at line 330

Reply:
this print should be only in the case of the default parallel-jobs where user is not using --parallel-jobs which is this condition - (parallelImportJobs == -1), how can we remove it from both the places add at some other?

Reply:
resolved, over discussion.